lib/erb_lint/cli.rb

```diff
@@ -39,9 +39,9 @@ def run(args = ARGV)
       load_config
 
       if !@files.empty? && lint_files.empty?
-        success!("no files found...\n")
+        failure!("no files found...\n")
       elsif lint_files.empty?
-        success!("no files given...\n#{option_parser}")
+        failure!("no files given...\n#{option_parser}")
```

Comment:
nit: since we're here, I would actually just drop the "no files given" from the message. It makes it seems like the user _has_ to give files to continue, but in reality it's one of two options (either giving files, or having them discovered through config). This probably also contradicts the usage message.

What do you think?

Reply:
If we drop it completely, then there won't be a message that will be helpful for the failure. Maybe we can modify the message to say something like `specify files or config`?

Reply:
That works for me too, or `no files found or given` (to mirror the other message a little better).

I actually like a combination of both what you recommend and what we already have. We would tell the user the problem ("no files ....") and what they can do to resolve that problem ("specify files or config").